script/tool/lib/src/update_dependency_command.dart

```diff
@@ -181,6 +229,65 @@ ${response.httpResponse.body}
     return PackageResult.success();
   }
 
+  /// Handles all of the updates for [package] when the target dependency is
+  /// an Android dependency.
+  Future<PackageResult> _runForAndroidDependency(
+      RepositoryPackage package) async {
+    if (_targetAndroidDependency == 'gradle') {
+      print('${indentation}Updating examples running on Android...');
```

Comment:
Optional: I would remove this print since we have specific details below; I think this will end up being noise in the output.